### PR TITLE
integrated eos-icons-react library

### PR DESCRIPTION
**Pull request by mharshita**
Integrated eos-icons-react library

- [x] My code follows the code style of this project.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Review by kbhutani0001 — APPROVED** [review]

**Comment by kbhutani0001** [comment]
@mharshita one thing, can we also remove the eos-icons css if it is not being used anywhere? that will reduce the package size

**Comment by mharshita** [comment]
> @mharshita one thing, can we also remove the eos-icons css if it is not being used anywhere? that will reduce the package size

We cannot remove eos-icons css because of padding and margins

**Comment by kbhutani0001** [comment]
> > @mharshita one thing, can we also remove the eos-icons css if it is not being used anywhere? that will reduce the package size
> 
> We cannot remove eos-icons css because of padding and margins

what do you mean? i meant the font/css files which were being used for the icons. since we are using the react package now, those shouldnt be needed anymore right?

**Comment by mharshita** [comment]
> > > @mharshita one thing, can we also remove the eos-icons css if it is not being used anywhere? that will reduce the package size
> > 
> > 
> > We cannot remove eos-icons css because of padding and margins
> 
> what do you mean? i meant the font/css files which were being used for the icons. since we are using the react package now, those shouldnt be needed anymore right?

Oh, sorry. I got confused with eos-icons class in scss files😅.... I will remove the css files...

File: src/components/Form.jsx [hunk 1]
```diff
@@ -9,9 +10,7 @@ export const Input = ({ type, updateState }) => {
         type='password'
       >
         <div class='input-group-append'>
-          <i class='eos-icons eos-18 toggle-password js-toggle-password'>
-            visibility
-          </i>
+          <EOS_VISIBILITY class='eos-icons eos-18 toggle-password js-toggle-password' />
```
Comment (vinayaksh42) [inline src/components/Form.jsx:13]: `class` needs to be changed to `className`

**Pull request merged**